test/bdd/bddtests_test.go

```diff
@@ -61,7 +61,7 @@ func TestMain(m *testing.M) {
 					composition = append(composition, newComposition)
 				}
 				fmt.Println("docker-compose up ... waiting for containers to start ...")
-				testSleep := 15
+				testSleep := 5
```

Comment:
is 5s reliably enough time?

Reply:
I reverted my previous change where docker-compose command script was building and launching `aries-agentd`, now we are building image beforehand in new make targets I added, so start time of 5s is enough.